frontend/explorer/public/hash.html

```diff
@@ -23,7 +23,7 @@
 	</div>
 
 	<div class='banner'>
-		<h1 class='banner-head'>ThreeFold Chain Hash Explorer</h1>
```

Comment:
we never said to remove the class. The class should be kept for css purposes, you'll need to give it a unique ID name such as 'page-title'.